models/gas/ethereum/gas_ethereum_fees_traces.sql

```diff
@@ -0,0 +1,79 @@
+{{ config(
+    alias = 'fees_traces',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash', 'trace'],
+    )
+}}
+
+WITH traces AS (
+     SELECT traces.block_time
+     , traces.block_number
+     , traces.tx_hash
+     , MAX(traces.from) AS trace_from
+     , MAX(traces.to) AS trace_to
+     , traces.trace
+     , substring(MAX(traces.input),1,10) AS trace_method
+     , SUM(traces.gas_used) AS gas_used
+     FROM (
+          SELECT from
+          , to 
+          , tx_hash
+          , trace_address AS trace
```

Comment:
trace_type (i.e. call vs delegatecall) & trace_value (eth transferred) would be good to add as well